### PR TITLE
DOCU-2908 Endpoint deprecation note and instruction 

### PR DESCRIPTION
**Pull request by Guaris**
https://konghq.atlassian.net/browse/DOCU-2908

Adds instructions on how to make a single request to the vitals API. 
https://deploy-preview-5032--kongdocs.netlify.app/gateway/latest/kong-enterprise/analytics/
~~- New reference doc that lists the mapping between old and new endpoints.~~
~~- Example request to a new endpoint.~~
~~- Links to new doc in the navigation bar.~~
~~- Addition of a troubleshooting section to the analytics section.~~

### CODE REVIEW

File: app/_src/gateway/kong-enterprise/analytics/index.md [hunk 1]
```diff
@@ -31,7 +31,47 @@ When using Vitals, note:
   * If a user does not have access to Vitals data, charts will not display.
 
 ## Vitals API
-Vitals data is available via endpoints on Kong’s Admin API. Access to these endpoints may be controlled via Admin API RBAC. The Vitals API is described in the OAS (Open API Spec, formerly Swagger) file. See a sample here (downloadable file): [`vitals.yaml`](/api/vitals.yaml).
+
+Vitals data is available using Vitals API. Access to these endpoints can be managed using RBAC. The Vitals API is described using the Open API Specification, you can view it by downloading the [`vitals.yaml`](/api/vitals.yaml) file. You can use the API to retrieve health and traffic information from your cluster.
```
Comment (lena-larionova) [inline app/_src/gateway/kong-enterprise/analytics/index.md:35]:
```suggestion
Vitals data is available using Vitals API. Access to these endpoints can be managed using RBAC. 

The Vitals API is described using the Open API Specification. You can view it by downloading the [`vitals.yaml`](/api/vitals.yaml) file. Use the API to retrieve health and traffic information from your cluster.
```
Got rid of the second "you can" for better flow.

File: app/_src/gateway/kong-enterprise/analytics/index.md [hunk 2]
```diff
@@ -31,7 +31,47 @@ When using Vitals, note:
...
+
+For example, you can use the API to retrieve the number of requests for a service in a workspace by issuing the following `GET` request: 
+
+```bash
+curl -i -X GET http://localhost:8001/WORKSPACE_NAME/vitals/status_codes/by_service \
+    -d "service_id=030f4ade-69ed-48b0-b3e1-cd088a1f2a09" \
+    -d "interval=minutes"
+```
+
+A succesful request will return a payload with the following data:
```
Comment (Guaris) [inline app/_src/gateway/kong-enterprise/analytics/index.md:45]:
```suggestion
A successful request will return a payload with the following data:
```

File: app/_src/gateway/kong-enterprise/analytics/index.md [hunk 3]
```diff
@@ -31,7 +31,47 @@ When using Vitals, note:
...
+
+```bash
```
Comment (lena-larionova) [inline app/_src/gateway/kong-enterprise/analytics/index.md:47]:
```suggestion
```json
```

File: app/_src/gateway/kong-enterprise/analytics/index.md [hunk 4]
```diff
@@ -31,7 +31,47 @@ When using Vitals, note:
...
+
+For example, you can use the API to retrieve the number of requests for a service in a workspace by issuing the following `GET` request: 
+
+```bash
+curl -i -X GET http://localhost:8001/WORKSPACE_NAME/vitals/status_codes/by_service \
+    -d "service_id=030f4ade-69ed-48b0-b3e1-cd088a1f2a09" \
+    -d "interval=minutes"
+```
+
+A successful request will return a payload with the following data:
```
Comment (lena-larionova) [inline app/_src/gateway/kong-enterprise/analytics/index.md:45]:
```suggestion
A successful request returns a payload with the following data:
```

File: app/_src/gateway/kong-enterprise/analytics/index.md [hunk 5]
```diff
@@ -31,7 +31,49 @@ When using Vitals, note:
...
+
+Vitals data is available using Vitals API. Access to these endpoints can be managed using RBAC. 
+
+The Vitals API is described using the Open API Specification. You can view it by downloading the [`vitals.yaml`](/api/vitals.yaml) file. Use the API to retrieve health and traffic information from your cluster.
+
+For example, you can use the API to retrieve the number of requests for a service in a workspace by issuing the following `GET` request: 
+
+```bash
+curl -i -X GET http://localhost:8001/WORKSPACE_NAME/vitals/status_codes/by_service \
+    -d "service_id=030f4ade-69ed-48b0-b3e1-cd088a1f2a09" \
```
Comment (sichvoge) [inline app/_src/gateway/kong-enterprise/analytics/index.md:43]: Are we usually replacing "real" values with some placeholders here? I am not sure.

**Review by sichvoge — APPROVED** [review]
Only a small question but otherwise this is looking good for me.

File: app/_src/gateway/kong-enterprise/analytics/index.md (hunk 5 above)
Comment (Guaris) [reply app/_src/gateway/kong-enterprise/analytics/index.md:43]: We are moving in that direction to signal to people where they would replace their values, and also to make it clear that the code can't just be copied and ran.

File: app/_src/gateway/kong-enterprise/analytics/index.md [hunk 6]
```diff
@@ -31,7 +31,49 @@ When using Vitals, note:
   * If a user does not have access to Vitals data, charts will not display.
 
 ## Vitals API
-Vitals data is available via endpoints on Kong’s Admin API. Access to these endpoints may be controlled via Admin API RBAC. The Vitals API is described in the OAS (Open API Spec, formerly Swagger) file. See a sample here (downloadable file): [`vitals.yaml`](/api/vitals.yaml).
+
+Vitals data is available using Vitals API. Access to these endpoints can be managed using RBAC.
```
Comment (lena-larionova) [inline app/_src/gateway/kong-enterprise/analytics/index.md:35]:
```suggestion
Vitals data is available using the Vitals API. Access to these endpoints can be managed using RBAC. 
```

**Review by lena-larionova — APPROVED** [review]

**Pull request merged**